api/v1/models/AttendeeProject.js

```diff
@@ -0,0 +1,35 @@
+var _ = require('lodash');
+
+var Model = require('./Model');
+var AttendeeProject = Model.extend({
```

Comment:
Partially. The create track will use this model, but it will also allow us to accept suggestions for projects that people would like to see at the event.